modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -5,9 +5,9 @@
 [id="knative-serving-concurrent-autoscaling-requests_{context}"]
 = Configuring concurrent requests for Knative Serving autoscaling
 
-You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` field in the revision template.
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` spec in the revision template.
 
-Here is an example of `target` being used in a revision template:
+Here is an example of the `target` annotation being used in a revision template:
```

Comment:
The two "Here is an example of..." style sentences in this module could potentially become titles for the code blocks, if you wanted. E.g.:

```
.`target` annotation used in a revision template
[source,yaml]
----
apiVersion: serving.knative.dev/v1
...
```

Reply:
Nice suggestion, thank you!